### PR TITLE
Fix dd node filter bug

### PR DESCRIPTION
**Pull request by sunby**
Signed-off-by: sunby <bingyi.sun@zilliz.com>

### CODE REVIEW

File: internal/datanode/flow_graph_dd_node.go [hunk 1]
```diff
@@ -89,9 +89,9 @@ func (ddn *ddNode) Operate(in []flowgraph.Msg) []flowgraph.Msg {
 				if ddn.filterFlushedSegmentInsertMessages(msg.(*msgstream.InsertMsg)) {
 					continue
 				}
+			} else {
+				iMsg.insertMessages = append(iMsg.insertMessages, msg.(*msgstream.InsertMsg))
 			}
```
Comment (XuanYang-cn) [inline internal/datanode/flow_graph_dd_node.go:92]: We Dont need a else here

**Review by XuanYang-cn — CHANGES_REQUESTED** [review]
Adding else will filter all msg older than filterTime

**Review by XuanYang-cn — APPROVED** [review]
/lgtm
/approve

**Review by neza2017 — COMMENTED** [review]
/approve

**Pull request merged**